server/data/data-api/src/main/java/org/apache/james/mailrepository/api/MailRepositoryStore.java

```diff
@@ -60,6 +60,19 @@ default MailRepository create(MailRepositoryUrl url) throws MailRepositoryStoreE
      */
     Stream<MailRepositoryUrl> getUrls();
 
+    /**
+     * Return a {@link Stream} which contains all paths of the selected
+     * {@link MailRepository}'s
+     *
+     * @return paths

```

Comment:
What's the added value of this @return statement compared to method signature? (akka you can remove it, it's duplicated info)